packages/client/hmi-client/src/petrinet/petrinet-renderer.ts

```diff
@@ -329,6 +335,11 @@ export class PetrinetRenderer extends BasicRenderer<NodeData, EdgeData> {
 		});
 
 		this.on('node-click', (_eventName, _event, selection: D3SelectionINode<NodeData>) => {
+			// Set focus on node:
+			this?.chart?.selectAll('.node-ui').style('opacity', '0.3');
+			this?.chart?.selectAll('.edge').style('opacity', '0.3');
+			selection.style('opacity', '1');
```

Comment:
Easier to read if you pass in numbers `0.3`, `1.0` or whatnot.

The optional-chaining is weird .... why do you have `this?`?